inc/functions.inc

```diff
@@ -4922,6 +4922,13 @@ if (!function_exists("alehinta")) {
         }
       }
 
+      // Tarkistetaan viel� luetaanko piiri asiakkaalta vaiko tilaukselta
+      $piiricheck = t_avainsana("PIIRI_ALENNUS", "", "and selitetark = 'T'", "", "", "selitetark");
+
+      if ($piiricheck and !empty($laskurow['piiri'])) {
```

Comment:
@tonyselkamo Jos asiakkaan piirin haluu "nollata" valitsemalla laskulla "Ei piiriä", niin se keissi ei toimi tässä. Onko ongelma? Tuo iffi voisi olla muodossa:

if ($piiricheck and isset($laskurow['piiri'])) {

niin laskulle valittu tyhjä piiri - keissikin toimii.

Reply:
@juppe Joo totta, mäpäs vaihan tuon.